src/functions.php

```diff
@@ -322,7 +330,7 @@ function runLocally($command, $timeout = 60)
 
     $output = $process->getOutput();
 
-    if (isDebug() && !empty($output)) {
+    if (!isDebug() && !empty($output)) {
```

Comment:
Now this will be printing output all the time. Lets delete this block completely.